package.json

```diff
@@ -262,7 +263,10 @@
         "test": "npm run compile && node ./dist/test/index.js",
         "lint": "gulp lint",
         "build-plugin": "gulp build-plugin",
-        "vscode:prepublish": "webpack --mode production"
+        "vscode:prepublish": "webpack --mode production",
+        "download-api": "vscode-dts dev -f",
```

Comment:
Temporarily need them since the API definition of `1.59.0` has not yet published to NPM registry